src/pages/ProductDetail/ProductData/ProductData.js

```diff
@@ -2,6 +2,20 @@ import React from 'react';
 import styled from 'styled-components';
 
 function ProductData({ detailData }) {
+  let starCounting = '';
+
+  if (detailData.productLevel === 1) {
+    starCounting = '★☆☆☆☆';
+  } else if (detailData.productLevel === 2) {
+    starCounting = '★★☆☆☆';
+  } else if (detailData.productLevel === 3) {
+    starCounting = '★★★☆☆';
+  } else if (detailData.productLevel === 4) {
+    starCounting = '★★★★☆';
+  } else if (detailData.productLevel === 5) {
+    starCounting = '★★★★★';
+  }
```

Comment:
```suggestion
  const {productLevel} = detailData;
  const rating = "★".repeat(productLevel) + "☆".repeat(5-productLevel);
```

이런식으로 별점을 구성할 수 있을 것 같습니다